test/support/helper.js

```diff
@@ -1,14 +1,35 @@
 import chai from 'chai';
 import chaiEnzyme from 'chai-enzyme';
-import jsdom from 'jsdom';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { JSDOM } from 'jsdom';
 import path from 'path';
 import requirePaths from 'app-module-path';
 
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+const { window } = jsdom;
+
+function copyProps(src, target) {
+  const props = Object.getOwnPropertyNames(src)
+    .filter(prop => typeof target[prop] === 'undefined')
+    .reduce((result, prop) => ({
+      ...result,
+      [prop]: Object.getOwnPropertyDescriptor(src, prop),
+    }), {});
+  Object.defineProperties(target, props);
+}
+
+global.window = window;
+global.document = window.document;
+global.navigator = {
+  userAgent: 'node.js',
+};
+copyProps(window, global);
```

Comment:
What is this for?

Reply:
This is black magic :black_joker:  copied from: https://github.com/airbnb/enzyme/blob/master/docs/guides/jsdom.md

The reason was that due to the package update there were significant changes and the previous code didn't work. As far as I understand it, it will populate the props with the JSDOM stuff. The user agent is probably not required but I don't know the inner workings of Enzyme and I don't think it will matter.